developer/requirements.md

```diff
@@ -29,14 +29,12 @@ apt-get install -y --no-install-recommends curl ca-certificates bzip2 git build-
 
 Windows installation has a few more requirements:
 
-- [Node.js](https://nodejs.org/) with NPM
-- Win32 OpenSSL ([windows installer](https://slproweb.com/products/Win32OpenSSL.html))
-- Visual Studio 2008 redistributable
```

Comment:
Has this changed? It used to be this very specific and very old redistributable.